release-notes.txt

```diff
@@ -0,0 +1,208 @@
+BISDN Linux
+v4.6.0 Release Notes
+Date: 2022-05-03
+
+! New in this release
+- New platform Accton-AS5835-54x is now supported
+- baseboxd: bump version to 1.11.2
+- Licenses in build process were updated to use the correct SPDX identifier
+- Linux kernel updated to version 5.15.35
+- OF-DPA updated to support configuraton of MPLS pseudowires using RYU
+- FRR updated to version 8.2
+- baseboxd/OF-DPA: support added for creating and enabling Broadcom KNET interfaces
+
+! Bug fixes
+- sysctl max read buffer size was set lower than the defaul read buffer size, causing baseboxd to intitialize with a buffer size that is too small. This could lead to loss of netlink messages, causing an out of sync switch state
+- Broadcom-KNET interfaces: now properly generates random MAC addresses
+- Broadcom-KNET interfaces: fixed issue related to race condition upon interface creation
+- baseboxd: broadcom-KNET interfaces are now properly removed when stopping baseboxd
+- client_drivshell: Fixed an issue when running that caused client_drivshell to segfault when running 'client_drivshell log'
+
+Changes between v4.5.0 and v4.6.0:
+
+bisdn-onie-additions:
+mk_bisdn_yocto_image.sh: add note on legacy status
+mk_bisdn_yocto_image: document usage
+mk_bisdn_yocto_image: document source
+mk_onie2: add rudimentary code documentation
+accton-as4610/platform.conf: add coments
+accton-as4610/platform.conf: document source
+u-boot-arch/install.sh: add comments
+u-boot-arch/install.sh: document source
+x86_64/install.sh: document source
+installer/install.sh: document functions
+installer/install.sh: add comments
+installer: accton-as4610/platform.conf: fix default configuration
+installer: fix detect_bisdn_linux_msdos_partition
+installer: fix create_bisdn_linux_msdos_partition args
+accton-as5835-54x: add platform.conf
+
+build-bisdn-linux:
+accton-as5835-54x: add platform to local.conf.sample
+release-notes: added release notes for v4.5.0 (#44)
```

Comment:
While technically true, we may want to drop this line.

Reply:
I removed it